internal/state/test/factory/block.go

```diff
@@ -38,15 +38,31 @@ func MakeBlocks(n int, state *sm.State, privVal types.PrivValidator) []*types.Bl
 func MakeBlock(state sm.State, height int64, c *types.Commit) *types.Block {
 	block, _ := state.MakeBlock(
 		height,
-		factory.MakeTenTxs(state.LastBlockHeight),
+		MakeData(factory.MakeTenTxs(state.LastBlockHeight), nil, nil),
 		c,
-		nil,
-		nil,
 		state.Validators.GetProposer().Address,
 	)
 	return block
 }
 
+func MakeData(txs []types.Tx, evd []types.Evidence, msgs []types.Message) types.Data {
```

Comment:
Aren't this duplicate of `types.test_util.go` ? Or you can't access that one in here ?

Reply:
I just didn't want to export it from the types package, as it is only a test util function. You're correct tho, keeping only one copy in types testutil would eliminate the need for one here